doc/cephfs/fs-volumes.rst

```diff
@@ -260,6 +260,31 @@ List subvolumes using::
 
 .. note:: subvolumes that are removed but have snapshots retained, are also listed.
 
+Set custom metadata on the subvolume as a key-value pair using::
+
+    $ ceph fs subvolume metadata set <vol_name> <subvol_name> <key_name> <value> [--group_name <subvol_group_name>]
+
+.. note:: If the key_name already exists then the old value will get replaced by the new value.
+
+.. note:: key_name and value should be string of ASCII characters with arbitrary lenght. key_name is case-insensitive and always stored in lower case.
```

Comment:
```
key_name and value should be a string of ASCII characters (as specified in python's string.printable).
```
No need to mention that it should be of arbitrary length.

Interesting, what makes the key_name case-insensitive? The value is case-sensitive?

Reply:
This is behaviour by config-parser in python: https://docs.python.org/3/library/configparser.html
```
Note also that keys in sections are case-insensitive and stored in lowercase
```

Reply:
Thanks for the explanation.